ui/src/virtualconsole/vccuelist.cpp

```diff
@@ -112,8 +114,11 @@ VCCueList::VCCueList(QWidget* parent, Doc* doc) : VCWidget(parent, doc)
     m_linkCheck = new QCheckBox(tr("Link"));
     grid->addWidget(m_linkCheck, 1, 0, 1, 2, Qt::AlignVCenter | Qt::AlignLeft);
 
+    QFontMetrics m_fm = QFontMetrics(this->font());
+    
     m_sl1TopLabel = new QLabel("100%");
     m_sl1TopLabel->setAlignment(Qt::AlignHCenter);
+    m_sl1TopLabel->setFixedWidth(m_fm.width("100%",-1));
```

Comment:
You don't actually need to pass -1 as argument as it is the default.
C++ allows to omit them.
Same below